frontend/src/resources/submariner-config.ts

```diff
@@ -69,4 +71,5 @@ export const submarinerConfigDefault: SubmarinerConfigDefaults = {
     azureInstanceType: 'Standard_F4s_v2',
     openStackInstanceType: 'PnTAE.CPU_4_Memory_8192_Disk_50',
     loadBalancerEnable: false,
+    brokerGlobalnetCIDR: '242.0.0.0/8', //NOSONAR
```

Comment:
Is NOSONAR required here?

Reply:
Yes ,since the value is a an IP , Sonar thinks it is a hard-coded IP and marks it as an vulnarability .

Reply:
I did not know about the //NOSONAR comment. You can also mark it as safe in SonarCloud.